apps/docs/src/packages/storybook-utils.md

```diff
@@ -0,0 +1,106 @@
+---
+outline: [2, 3]
+---
+
+<script lang="ts" setup>
+import packageJson from "../../../../packages/storybook-utils/package.json";
+</script>
+
+# @sit-onyx/storybook-utils
+
+{{ packageJson.description }}.
+
```

Comment:
Should we add some info like
```suggestion
Required Storybook version 7.6 or higher

```

Reply:
I thinks its sufficient that its defined in the peerDependencies.
The package works with all versions of storybook >= 7